src/model/encoding/__tests__/convertFromRawToDraftState-test.js

```diff
@@ -16,19 +16,120 @@ jest.disableAutomock();
 
 const convertFromRawToDraftState = require('convertFromRawToDraftState');
 
-describe('convertFromRawToDraftState', () => {
-  it('must map falsey block types to default value of unstyled', () => {
-    const rawState = {
-      blocks: [
-        {text: 'AAAA'},
-        {text: 'BBBB', type: null},
-        {text: 'CCCC', type: undefined},
-      ],
-      entityMap: {},
-    };
-
-    const contentState = convertFromRawToDraftState(rawState);
-    const blockMap = contentState.getBlockMap();
-    blockMap.forEach(block => expect(block.type).toEqual('unstyled'));
-  });
+const assertDraftState = (rawState, expected) => {
+  // https://github.com/facebook/draft-js/pull/1490
+  // TODO: mitermayer - re-enable snapshot testing after PR [1490] is merged
+  expect(
+    convertFromRawToDraftState(rawState)
+      .getBlockMap()
+      .toJS(),
```

Comment:
By adding the `.toJS()` we can be less fragile to snapshot assertions accross different immutable dependencies